test/signup.test.js

```diff
@@ -0,0 +1,59 @@
+const chai = require('chai');
+const app = require('../index');
+const request = require('supertest')(app);
+const db = require('../models/');
+const mocha = require('mocha');
+const faker = require('faker');
+
+const describe = mocha.describe;
+
+const expect = chai.expect; 
+const user =  {
+    email: faker.internet.email(),
+    username:faker.name.lastName(),
+    password: faker.internet.password(),
+};
+
+
+describe('POST requests', function() {
+    beforeEach(function(done){
+      db.User.destroy({
+          where: {},
+      }).then(user => console.log(user));
+      done();
+        
+    });
+     describe('/api/users/', function(){
+
+        describe('status code', function(){
+            it('should be equal to 201', function(done){
+                request.post('/api/users/')
+                .send({user: user})
+                .expect(201, done);
+            });
+        })
+        describe('request body', function(){
+            it('should have a data property that contains a token string property', function(done){
```

Comment:
`should have a token property`

Reply:
Do you mean that I should not put the token in the data property? What I have now is:
{
   data:{
     username: 'username',
    "token": "eycodac dscds",
    ...
  }
}

Reply:
should I change it to

{
  data:{
         username: 'username',
         ...
   },
  "token": "eycodac dscds",
   .....
}

Reply:
change ~should have a data property that contains a token string~ to **should have a token property**